AbletonMCP_Remote_Script/handlers/mixer.py

```
@@ -0,0 +1,300 @@
+    except Exception as e:
+        if ctrl:
+            ctrl.log_message("Error setting track pan: " + str(e))
+        raise
+
+
+def set_track_mute(song, track_index, mute, ctrl=None):
+    """Set track mute state."""
+    try:
+        track = get_track(song, track_index)
+        track.mute = mute
+        return {"track_index": track_index, "mute": track.mute}
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)